metadata/src/main/java/org/apache/kafka/controller/ActivationRecordsGenerator.java

```diff
@@ -165,6 +165,8 @@ static ControllerResult<Void> recordsForNonEmptyLog(
                         throw new RuntimeException("Should not have ZK migrations enabled on a cluster that was " +
                             "created in KRaft mode.");
                     }
+                    logMessageBuilder
+                            .append("This is expected because this is a de-novo KRaft cluster.");
```

Comment:
nit: remove the unnecessary line break here

Reply:
@mumrah , i make new commit to apply your comments. 
However, the `CI` still is failed. 
The changes which i made seems not to be related with failure of CI. 
Am i wrong? 
<img width="673" alt="image" src="https://github.com/apache/kafka/assets/90125071/d013add9-814d-4ad9-8c2e-cb2955455645">